pds_doi_service/core/actions/check.py

```diff
@@ -30,7 +33,7 @@
 from pds_doi_service.core.util.emailer import Emailer
 from pds_doi_service.core.util.general_util import get_logger
 
-logger = get_logger('pds_doi_core.actions.check')
+logger = get_logger('pds_doi_service.core.actions.check')
```

Comment:
This comment is outside of the scope of the PR 169 but as an observation: the `get_logger` enforces a hard-coded logging format, meaning that a user can't override this for specific needs (XML logging, JSON logging, etc.) at run time. Sticking to the basic pattern of `logger = logging.getLogger(__name__)` is easier and more flexible.

Reply:
Yes we can have __name__ everywhere. @collinss-jpl found the code this way. It is one more code cleaning which can/should be done